pkg/rip7560/transaction/transaction_args.go

```diff
@@ -27,21 +27,36 @@ type TransactionArgs struct {
 	Input *hexutil.Bytes `json:"input"`
 
 	// Introduced by AccessListTxType transaction.
-	AccessList *types.AccessList `json:"accessList,omitempty" rlp:"nil"`
-	ChainID    *hexutil.Big      `json:"chainId,omitempty" rlp:"nil"`
+	AccessList *types.AccessList `json:"accessList,omitempty"`
+	ChainID    *hexutil.Big      `json:"chainId,omitempty"`
+
+	// For BlobTxType
+	//BlobFeeCap *hexutil.Big  `json:"maxFeePerBlobGas"`
+	//BlobHashes []common.Hash `json:"blobVersionedHashes,omitempty"`
+
+	// For BlobTxType transactions with blob sidecar
+	//Blobs       []kzg4844.Blob       `json:"blobs"`
+	//Commitments []kzg4844.Commitment `json:"commitments"`
+	//Proofs      []kzg4844.Proof      `json:"proofs"`
+
+	// This configures whether blobs are allowed to be passed.
+	//blobSidecarAllowed bool
 
 	// Introduced by RIP-7560 Transaction
-	Sender        *common.Address `json:"sender"`
-	Signature     *hexutil.Bytes  `json:"signature"`
-	Paymaster     *common.Address `json:"paymaster,omitempty"     rlp:"nil"`
-	PaymasterData *hexutil.Bytes  `json:"paymasterData,omitempty" rlp:"nil"`
-	Deployer      *common.Address `json:"deployer,omitempty"      rlp:"nil"`
-	DeployerData  *hexutil.Bytes  `json:"deployerData,omitempty"  rlp:"nil"`
-	BuilderFee    *hexutil.Big    `json:"builderFee"`
-	ValidationGas *hexutil.Uint64 `json:"verificationGasLimit"`
-	PaymasterGas  *hexutil.Uint64 `json:"paymasterVerificationGasLimit"`
-	PostOpGas     *hexutil.Uint64 `json:"paymasterPostOpGasLimit"`
-	BigNonce      *hexutil.Big    `json:"bigNonce"`
+	Sender            *common.Address `json:"sender"`
+	AuthorizationData *hexutil.Bytes  `json:"authorizationData,omitempty"`
+	ExecutionData     *hexutil.Bytes  `json:"executionData,omitempty"`
+	Paymaster         *common.Address `json:"paymaster,omitempty"`
+	PaymasterData     *hexutil.Bytes  `json:"paymasterData,omitempty"`
+	Deployer          *common.Address `json:"deployer,omitempty"`
+	DeployerData      *hexutil.Bytes  `json:"deployerData,omitempty"`
```

Comment:
Currently, rlp:‘nil’ is not required; this structure in the bundler is unified because geth's transaction_args is the original and we think it's best to follow the same format as much as possible.